src/pages/PersonalCalendar/components/TodaySchedule.tsx

```diff
@@ -1,24 +1,43 @@
-import { classSchedules } from '@/mockdata/scheduleData';
-import { MapPin } from 'lucide-react';
+import { useCalendarStore } from '@/store/calendar/useCalendarStore';
+import { getTimePart } from '@/utils/dateTimeUtils';
+import { Paperclip } from 'lucide-react';
+import { useEffect } from 'react';
+
 const TodaySchedule = () => {
+  const { todayEvents, getTodayEvents } = useCalendarStore();
+
+  useEffect(() => {
+    void getTodayEvents();
+  }, [getTodayEvents]);
+
+  useEffect(() => {
+    console.log('todayEvents: ', todayEvents);
+  }, [todayEvents]);
+
   return (
     <div className="px-3 mb-8">
       <h3 className="mb-4 text-lg font-semibold text-gray-800">오늘 일정</h3>
       <div className="space-y-3">
-        {classSchedules.map((schedule, index) => (
-          <div key={index} className="flex flex-row p-2 pl-0 bg-white rounded-lg">
-            <div className="flex justify-between items-center mr-4 mb-1">
-              <span className="text-sm font-medium text-mainBlue">{schedule.time}</span>
-            </div>
-            <div>
-              <p className="mb-1 text-sm font-medium text-[#1C398E]">{schedule.title}</p>
-              <p className="flex gap-1 items-center text-xs text-mainBlue">
-                <MapPin className="w-4 h-4" />
-                {schedule.location}
-              </p>
+        {todayEvents &&
+          todayEvents.map((events, index) => (
```

Comment:
```suggestion
        {todayEvents?.map((events, index) => (
```
모던 자바스크립트에서는 optional chaining을 쓰는게 깔끔합니다 😁